### PR TITLE
Add vumi dashboard

### PR DESCRIPTION
**Pull request by gsvr**
https://trello.com/c/HmRuJ1ON/79-8-rr-ussd-add-vumi-dashboard

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.  There are a few metrics still required that I'll add as I go - want to keep this issue size down.

File: config/config_smsapp.qa.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "sms-infohub-qa",
+    "env": "qa",
```
Comment (imsickofmaps) [inline config/config_smsapp.qa.json]: remove

File: src/utils.js [hunk 2]
```diff
@@ -209,5 +209,29 @@ go.utils = {
...
+    },
+
+    fire_returning_user_metrics: function(im, contact) {
+        var today = go.utils.get_today(im.config);
+        contact.extra.last_seen = today;
+        var user_registered = contact.extra.status === 'refugee' || contact.extra.status === 'migrant';
+        var no_last_returning_metric = contact.extra.last_returning_metric_fire === undefined;
+        var old_last_returning_metric = go.utils.date_difference(today, contact.extra.last_returning_metric_fire) > 7;
+
+        if (user_registered && (no_last_returning_metric || old_last_returning_metric)) {
```
Comment (imsickofmaps) [inline src/utils.js:227]: Please just add a comment to explain what this does. We know now, but in 3 months we might go "eh?"

**Comment by imsickofmaps** [comment]
Minor comments then :+1:

**Pull request merged**